Mathlib/Order/Filter/AtTopBot/Basic.lean

```diff
@@ -441,7 +441,6 @@ theorem tendsto_add_atTop_iff_nat {f : ℕ → α} {l : Filter α} (k : ℕ) :
 
 theorem map_div_atTop_eq_nat (k : ℕ) (hk : 0 < k) : map (fun a => a / k) atTop = atTop :=
   map_atTop_eq_of_gc (fun b => k * b + (k - 1)) 1 (fun _ _ h => Nat.div_le_div_right h)
-    -- Porting note: there was a parse error in `calc`, use `simp` instead
```

Comment:
Flagging this change for a second opinion

Reply:
The porting note seems to be outdated since the `simp` was removed in #16611.